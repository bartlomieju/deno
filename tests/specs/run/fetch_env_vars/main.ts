Deno.serve({
  handler(req) {
    console.log(req.headers);
    Deno.exit(0);
  },
  onListen(addr) {
    const url = `http://${addr.hostname}:${addr.port}/`;
    fetch(url);
  },
  port: 0,
});
