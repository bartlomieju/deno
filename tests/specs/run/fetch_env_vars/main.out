Headers {
[WILDCARD]
  "cdn-loop": "land",
[WILDCARD]
  "x-deno-fetch-token": "deno"
}
